profiles/package.mask

```diff
@@ -1,3 +1,7 @@
+# Wolfgang E. Sanyer <WolfgangESanyer@gmail.com> (2021-01-02)
+# Last release in 2018, package does not appear to compile properly.
```

Comment:
Can you add a hint why it fails? Is it recent `cabal` incompatibility or missing instance declarations?